src/ic-cdk/src/api/management_canister/bitcoin/types.rs

```diff
@@ -34,10 +34,10 @@ pub type BitcoinAddress = String;
 /// Block Hash.
 pub type BlockHash = Vec<u8>;
 
-/// Element in the Response of [bitcoin_get_current_fee_percentiles](super::bitcoin_get_current_fee_percentiles).
+/// Element in the Response of [`bitcoin_get_current_fee_percentiles`](super::bitcoin_get_current_fee_percentiles).
 pub type MillisatoshiPerByte = u64;
 
-/// Identifier of [Utxo].
+/// Identifier of [`Utxo`].
```

Comment:
Were these backticks from auto-formatting?

I want to apply the non-backticks style in this project.

Reply:
No, I added them; there is a clippy::pedantic lint against code elements outside backticks.

Why put code elements in the prose font instead of the code font?

Reply:
I see. I had an impression that my code editor automatically removes the backticks in brackets. But that seems to be wrong.

I agree code font is better here.

I'll apply the new rule to the doc comments later.